odoo/addons/test_new_api/tests/test_many2many.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from odoo.tests.common import TransactionCase
+
+
+class Many2manyCase(TransactionCase):
+    def setUp(self):
```

Comment:
`setUpClass` instead of `setUp`? It will use savepoint instead of creating the same data for each test.

Reply:
`TransactionCase` does not use a savepoint in version 14.0 yet :smile: